pipelines/cf-operator-testing-image/pipeline.yml

```diff
@@ -82,5 +82,5 @@ jobs:
         DOCKER_TEAM_USERNAME: ((dockerhub.username))
         DOCKER_TEAM_PASSWORD_RW: ((dockerhub.password))
         REGISTRY_NAMESPACE: "cfcontainerization"
-      file: ci/pipelines/release-images-dev/tasks/build.yml
+      file: ci/pipelines/cf-operator-testing-image/tasks/build.yml
```

Comment:
This task is used also from quarks-gora, shall we move it in common tasks instead?